### PR TITLE
Migrated to Navigation Component

### LINKED ISSUE #664: Migrate to the Navigation component
Needs to migrate the navigation flow of screens to the Navigation component. The bets option to build navigation between screens. Reference: https://developer.android.com/guide/navigation

To do this, needs to change the app to single Activity and handle the Fragment transactions in the Activity.

### PR DESCRIPTION
**Pull request by amiapmorais**
fix #664 

## Type of changes

- [x] **Refactor**
---

## Checklist:

- [x] My code is understandable
- [ ] My code follows the styleguide of this project
- [ ] My changes don't generate new warnings/errors
- [x] I have performed a self-review of my own code (build/lint/tests)
- [x] **No tests required.**

image

### CODE REVIEW

**Review by roandersonpinheiro — CHANGES_REQUESTED** [review]

File: android-client/Espoo/app/src/main/java/com/espoo/android/view/LoginFragment.kt [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            val password = binding.editTextPassword.text.toString()
+            if (!validator.validateInputNotEmpty(userName)) {
+                Toast.makeText(context,
+                    "${getString(R.string.login_username)} ${getString(R.string.field_does_not_filled)}",
+                    Toast.LENGTH_LONG).show()
+            } else if (!validator.validateInputNotEmpty(password)) {
+                Toast.makeText(context,
+                    "${getString(R.string.login_password)} ${getString(R.string.field_does_not_filled)}",
+                    Toast.LENGTH_LONG).show()
+            } else {
```
[the remark this answers is not in the document]
Comment (amiapmorais) [reply android-client/Espoo/app/src/main/java/com/espoo/android/view/LoginFragment.kt:59]: Thank you for your reviews @roandersonpinheiro. I'm not totally familiarized with the MVVM arquiteture, working with ViewModel classes. I thought I could implement the MVVM arquiteture later in the project, but maybe it's a good time to do this. I will create a specific issue to this and start to working this week.

[the remark this answers is not in the document]
Comment (roandersonpinheiro) [reply android-client/Espoo/app/src/main/java/com/espoo/android/view/LoginFragment.kt:59]: Okay Ami, good job. If you need help I can help you with that.

[the remark this answers is not in the document]
Comment (amiapmorais) [reply android-client/Espoo/app/src/main/java/com/espoo/android/view/LoginFragment.kt:59]: I'm very grateful. If a need it I'll ask for help, you bet. And when I done I will ask your review anyway.

[the remark this answers is not in the document]
Comment (roandersonpinheiro) [reply android-client/Espoo/app/src/main/java/com/espoo/android/view/LoginFragment.kt:59]: Ok @amiapmorais , we can create another task to refactor the code and think of an architecture to implement, what do you think?

[the remark this answers is not in the document]
Comment (amiapmorais) [reply android-client/Espoo/app/src/main/java/com/espoo/android/view/LoginFragment.kt:60]: Let's do that.

**Pull request merged**